github/messages.go

```diff
@@ -117,3 +147,42 @@ func validateSignature(signature string, payload, secretKey []byte) error {
 	}
 	return nil
 }
+
+// ParseWebHookType returns the type of payload
+func ParseWebHookType(r *http.Request) string {
```

Comment:
In this case, I wouldn't call this func parse.

Parse is a good name for something that can fail, and the signature usually has (Something, error) form.

Here, you're really just getting the type of payload from the request headers.

How about renaming it to just `WebHookType(r *http.Request)`? It feels like leaving out `Get` is better.

```
// WebHookType returns the event type of webhook request r.
func WebHookType(r *http.Request) string {
```

If not that, then call it `GetWebHookType`, but `ParseWebHookType` is definitely not the best name.